packages/get-packages/README.md

```diff
@@ -1,8 +1,8 @@
 # @manypkg/get-packages
 
-> A simple utility to get the packages from a monorepo, whether they're using Yarn, Bolt, Lerna, pnpm or Rush
+> A simple utility to get the packages from a monorepo, whether they're using Yarn, npm, Lerna, pnpm or Rush
```

Comment:
Is npm monorepos actually supported in the package? IIUC since it uses `package.json#workspaces` it may be incorrectly detected as yarn.

Reply:
good catch, https://github.com/Thinkmill/manypkg/pull/250